deploy/012-deploy-prime.ts

```diff
@@ -77,7 +80,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const xvsVaultAlphaNumerator = 1;
   const xvsVaultAlphaDenominator = 2;
   const loopsLimit = 20;
-  const isTimeBased = false; // revise this value when deploying on L2s
+  const isTimeBased = true; // revise this value when deploying on L2s
```

Comment:
Wouldn't this be less error-prone?

```suggestion
  const isTimeBased = blocksPerYear[network.name] === 0;
```